src/material_interface.cpp

```diff
@@ -614,6 +615,12 @@ bool material_interface(
                 material_cells.emplace_back(1);
                 material_cells.back()[0] = i;
```

Comment:
Do we have information about the orientation of a shell with respect to a material cell? Can a shell be shared by two neighboring material cells?